cscs-checks/mch/collectives_halo.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, variant, bench_reference):
             self.num_tasks = 4
             self.num_gpus_per_node = 1
             self.num_tasks_per_node = 1
-            self.modules = ['craype-accel-nvidia60', 'CMake']
+            self.modules = ['craype-accel-nvidia60', 'CMake', 'cdt-cuda']
```

Comment:
Since you change the test, you can also add the parameter in the base class I think.